config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -45,7 +45,8 @@ presubmits:
         - --gcp-nodes=4
         - --gcp-zone=us-west1-b
         - --provider=gce
-        - --test_args=--ginkgo.focus=\[sig-network\]|\[Conformance\]|\[Feature:NetworkPolicy\] --ginkgo.skip=\[Feature:(Networking-IPv6|Example|Federation|PerformanceDNS)\]|DualStack|GCE|SCTP|Disruptive|Serial
+        # Skipping snat tests probably GCE related? https://github.com/kubernetes/test-infra/issues/20321
+        - --test_args=--ginkgo.focus=\[sig-network\]|\[Conformance\]|\[Feature:NetworkPolicy\] --ginkgo.skip=\[Feature:(Networking-IPv6|Example|Federation|PerformanceDNS)\]|DualStack|GCE|SCTP|Disruptive|Serial|SNAT
```

Comment:
finally, I´ve found the reason why the job was taking so long, we have to skip those
`LB.health.check|LoadBalancer|load.balancer|` 
https://github.com/kubernetes/test-infra/blob/ebc5cb37dcae724901842a6f4c67001f6113e135/config/jobs/kubernetes/sig-network/sig-network-kind.yaml#L142-L143

Reply:
wait, i dont see these failing ? why are we skpping them. ?

Reply:
```Antonio Ojea  remove the GCE LB tests, those take 900 seconds or more

```

Reply:
ah ok, its just for performance

Reply:
because each of them take more than 600 seconds, and those are gce specific ... the e2e framework is trying to be agnostic, if sometime they are removed they will break the job and you´ll spend some time to realise why

Reply:
that depend on GCE, if something breaks between the e2e and gce we can´t do anything, and they have their own tests
https://testgrid.k8s.io/sig-network-ingress-gce-e2e#ingress-gce-e2e

Reply:
pushed